accounts/abi/pack_test.go

```diff
@@ -42,7 +42,7 @@ func TestPack(t *testing.T) {
 		{
 			"uint8[]",
 			[]uint8{1, 2},
-			common.Hex2Bytes("000000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000002"),
+			common.Hex2Bytes("0000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000002"),
 		},
```

Comment:
for all these lines only `0000000000000000000000000000000000000000000000000000000000000020` has been added in the beginning